config/data_funcs.go

```diff
@@ -196,3 +198,39 @@ outer:
 	log.Debugf("Input obj %v", result)
 	return result
 }
+
+// GetCmdLineValues return a list of all values for a key (used for non rancher. boot cmdline)
+// OK, so the kernel boot cmdline is not unicode
+func GetCmdLineValues(cmdline, key string) []string {
```

Comment:
We already have an existing function for parsing command line values. It would be better to reuse that for consistency.